var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -133,7 +133,7 @@ class Acts(CMakePackage, CudaPackage):
     variant('pythia8', default=False, description='Build the Pythia8-based examples')
 
     # Build dependencies
-    # FIXME: Use spack's autodiff package once there is one
+    depends_on('autodiff @5.11:', when='@4.10: -autodiff')
```

Comment:
```suggestion
    depends_on('autodiff @5.11:', when='@1.02: +autodiff')
```
Autodifff support has been here since ACTS v1.2, and you want to add the dependency when the `autodiff` variant is enabled.